package.json

```diff
@@ -21,11 +20,18 @@
     "url": "https://github.com/navitskyd/movie-FE/issues"
   },
   "homepage": "https://github.com/navitskyd/movie-FE#readme",
+  "browserslist": [
+    "last 2 versions"
+  ],
   "devDependencies": {
     "@babel/core": "^7.14.3",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/preset-env": "^7.14.4",
     "@babel/preset-react": "^7.13.13",
+    "@types/react": "^17.0.11",
+    "@types/react-dom": "^17.0.7",
+    "@types/react-redux": "^7.1.16",
+    "awesome-typescript-loader": "^5.2.1",
     "babel-loader": "^8.2.2",
     "babel-preset-react": "^6.24.1",
     "css-loader": "^5.2.6",
```

Comment:
where is less npm package?

Reply:
"dependencies": {
    "autoprefixer": "^10.2.6",
    "jest": "^27.0.4",
    "less-loader": "^9.0.0",